<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>RequestViews</class>
 <widget class="QDialog" name="RequestViews">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>400</width>
    <height>400</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <widget class="QPushButton" name="pushButton">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>10</y>
     <width>61</width>
     <height>25</height>
    </rect>
   </property>
   <property name="text">
    <string>Back</string>
   </property>
  </widget>
  <widget class="QLabel" name="label">
   <property name="geometry">
    <rect>
     <x>50</x>
     <y>90</y>
     <width>291</width>
     <height>20</height>
    </rect>
   </property>
   <property name="text">
    <string>You are not permitted to view this image :(</string>
   </property>
  </widget>
  <widget class="QTextEdit" name="quota_editText">
   <property name="geometry">
    <rect>
     <x>183</x>
     <y>140</y>
     <width>161</width>
     <height>41</height>
    </rect>
   </property>
  </widget>
  <widget class="QLabel" name="label_2">
   <property name="geometry">
    <rect>
     <x>50</x>
     <y>150</y>
     <width>131</width>
     <height>20</height>
    </rect>
   </property>
   <property name="text">
    <string>Views Requested:</string>
   </property>
  </widget>
  <widget class="QPushButton" name="request_button">
   <property name="geometry">
    <rect>
     <x>50</x>
     <y>210</y>
     <width>291</width>
     <height>25</height>
    </rect>
   </property>
   <property name="text">
    <string>Request</string>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
